### ISSUE TITLE
Touch friendly "gestures" (like in NewPipe / VLC mobile)

### ISSUE THREAD

**Issue by Dani3I** [igrigorik/videospeed#1102]
There are devices such as "surface pro" tablets which have touchscreen on desktop. There is also fork of this extension for firefox, and it is possible to install firefox extensions on mobile.

In app called NewPipe (for android) when video is in full-screen, touching on right portion of the screen and dragging finger up or down, changes volume (up or down). Touching on left portion of the screen and dragging up or down, changes brightness.
It would be great if this extension would allow to use this or similar gesture to change speed of the video. Simmilarly it also works in mobile version of VLC app.
Another way I thought of is to create some semi transparent button appears on middle left or right side (or other) and touching and dragging it would "reveal" slider to change video speed (and this dragging would change the video speed up or down). It would be kinda simmilar to current semi-transparent button, but it would open slider instead of buttons, and it would appear in different (than upper left) position of the screen. So it could be then made in settings to choose If someone wants buttons or slider; If button should be always visible or auto-hiding; What is the min and max speed. If the button would be only clicked (and not dragged) then slider would also pop up but just clicking in desired position on slider would change video speed (instead of dragging), and then I would argue that it would even benefit desktop users (but In my opinion even dragging would benefit desktop users - I really liked using scrolwheel to change video speed when it was avaiable).

**Comment by boognish-rising** [comment]
Yeah I'm responding from my Android tablet that I have Video Speed Controller installed on via Kiwi Browser. There's also Yandex and all of the many Firefox forks and a handful of other mobile browsers with extension support that could also install Video Speed Controller for use on a touch screen device. So I'm for this but I probably utilize Video Speed Controller on desktop more than anywhere if for no other reason then mobile video players generally have speed controls built in while this is not always the case on desktop so I would just want to make sure that the added functionality didn't detract from the desktop experience. In the case of apps like NewPipe, VLC, YT ReVanced or MX player etc., all of which have the type of touch gestures you're describing, interacting with these apps using a mouse is pretty smooth and intuitive. As long as the same could be said for Video Speed Controller if some this were to be added then I'm all for it
